src/System.Drawing.Common/tests/mono/System.Imaging/MetafileTest.cs

```diff
@@ -0,0 +1,552 @@
+            using (Metafile clone = (Metafile)mf.Clone())
+            {
+            }
+        }
+
+        static public void Check_MetaHeader_WmfPlaceable(MetaHeader mh)
+        {
+            Assert.Equal(9, mh.HeaderSize);
+            Assert.Equal(98, mh.MaxRecord);
+            Assert.Equal(3, mh.NoObjects);
+            Assert.Equal(0, mh.NoParameters);
+            Assert.Equal(1737, mh.Size);
+            Assert.Equal(1, mh.Type);
+            Assert.Equal(0x300, mh.Version);
+        }
+
+        public static void Check_MetafileHeader_WmfPlaceable(MetafileHeader header)
+        {
+            Assert.Equal(MetafileType.WmfPlaceable, header.Type);
+            Assert.Equal(0x300, header.Version);
+            // filesize - 22, which happens to be the size (22) of a PLACEABLEMETAHEADER struct
+            Assert.Equal(3474, header.MetafileSize);
+
+            Assert.Equal(-30, header.Bounds.X);
+            Assert.Equal(-40, header.Bounds.Y);
+            Assert.Equal(3096, header.Bounds.Width);
+            Assert.Equal(4127, header.Bounds.Height);
+            Assert.Equal(606, header.DpiX);
+            Assert.Equal(606, header.DpiY);
+            Assert.Equal(0, header.EmfPlusHeaderSize);
+            Assert.Equal(0, header.LogicalDpiX);
+            Assert.Equal(0, header.LogicalDpiY);
+
+            Assert.NotNull(header.WmfHeader);
+            Check_MetaHeader_WmfPlaceable(header.WmfHeader);
+
+            Assert.False(header.IsDisplay());
+            Assert.False(header.IsEmf());
+            Assert.False(header.IsEmfOrEmfPlus());
+            Assert.False(header.IsEmfPlus());
+            Assert.False(header.IsEmfPlusDual());
+            Assert.False(header.IsEmfPlusOnly());
+            Assert.True(header.IsWmf());
+            Assert.True(header.IsWmfPlaceable());
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void GetMetafileHeader_WmfPlaceable()
+        {
+            using (Metafile mf = new Metafile(Helpers.GetTestBitmapPath(WmfPlaceable)))
+            {
+                MetafileHeader header1 = mf.GetMetafileHeader();
+                Check_MetafileHeader_WmfPlaceable(header1);
+
+                MetafileHeader header2 = mf.GetMetafileHeader();
+                Assert.False(Object.ReferenceEquals(header1, header2));
+            }
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void GetMetafileHeader_FromFile_WmfPlaceable()
+        {
+            using (Metafile mf = new Metafile(Helpers.GetTestBitmapPath(WmfPlaceable)))
+            {
+                MetafileHeader header1 = mf.GetMetafileHeader();
+                Check_MetafileHeader_WmfPlaceable(header1);
+
+                MetaHeader mh1 = header1.WmfHeader;
+                Check_MetaHeader_WmfPlaceable(mh1);
+
+                MetaHeader mh2 = mf.GetMetafileHeader().WmfHeader;
+                Assert.False(Object.ReferenceEquals(mh1, mh2));
```

Comment:
You can instead use: `Assert.NotSame(mh1, mh2);` in all the places where you are doing `Assert.False(Object.ReferenceEquals(mh1, mh2));`

https://github.com/xunit/assert.xunit/blob/master/IdentityAsserts.cs#L18